README.md

```diff
@@ -229,6 +230,39 @@ In order to run the boxel-motion demo app:
 2. `pnpm start`
 3. Visit http://localhost:4200 in your browser
 
+## Payment Setup
+
+There is some pre-setup needed to enable free plan on development account:
+
+1. Use stripe cli to listen the webhook
+
+```
+stripe listen --forward-to localhost:4201/_stripe-webhook --api-key sk_test_api_key_from_the_sandbox_account
+```
+
+2. You will get webhook signing secret from stripe cli after Step 1 is done
+
+```
+> Ready! You are using Stripe API Version [x]. Your webhook signing secret is whsec_xxxxx
+```
+
+3. Set STRIPE_WEBHOOK_SECRET environment value in `packages/billing/stripe-webhook-handlers/index.ts` file
+
+```
+process.env.STRIPE_WEBHOOK_SECRET =
+  'whsec_xxxxx';
+```
+
+4. Go to `packages/realm-server` and run the following command to sync the stripe products to the database, make sure you have the stripe api key set
+
+```
+pnpm sync-stripe-products
```

Comment:
Will work without passing the ENV vars? I noted them in  https://github.com/cardstack/boxel/pull/1777#issuecomment-2490807673 ?